hack/setup-kind-in-ci.sh

```diff
@@ -91,10 +91,6 @@ data:
       addresses: ['${first_ip}-${last_ip}']
 LBCONFIGMAP
 
-# if [ -n "${ISTIO_VERSION}" ]; then
-# DOWNLOAD_ISTIO_VERSION_ARG="--istio-version ${ISTIO_VERSION}"
-# fi
-
 infomsg "Downloading istio"
 hack/istio/download-istio.sh
```

Comment:
you could uncomment what you deleted above and just pass in that env var here to support being able to run against a specific version:
```suggestion
hack/istio/download-istio.sh ${DOWNLOAD_ISTIO_VERSION}
```

Reply:
Remembering now why I commented this out... the download-istio script fails with a weird error when `DOWNLOAD_ISTIO_VERSION=""`.

Error:
```
Unknown argument []. Aborting.
```
Relevant place in script:
```
while [ $# -gt 0 ]; do
  key="$1"
  case $key in
    -div|--dev-istio-version)
      DEV_ISTIO_VERSION="$2"
      shift;shift
      ;;
    -iv|--istio-version)
      ISTIO_VERSION="$2"
      shift;shift
      ;;
    -o|--output)
      OUTPUT_DIR="$2"
      shift;shift
      ;;
    -h|--help)
      cat <<HELPMSG
Valid command line arguments:
  -div|--dev-istio-version <version>: If you want a dev version to download, use this option and do not use -iv.
                                      The value of this option is something like "1.4-dev" or "1.5-dev". If you
                                      want the latest master build, set this to "latest".
  -iv|--istio-version <#.#.#>: Released version of Istio to download. (default will be the latest version)
  -o|--output <dir> : Output directory where Istio is (or will be downloaded to if it doesn't exist).
  -h|--help : This message.
HELPMSG
      exit 1
      ;;
    *)
      echo "Unknown argument [$key]. Aborting."
      exit 1
      ;;
  esac
done
```

Seems like it's interpreting extra whitespace as an argument?

Reply:
that's odd. Wrap the arg value in quotes maybe? Not sure what github is doing weird when passing in the arguments.

Try `--istio-version "${DOWNLOAD_ISTIO_VERSION}"`